funnel/templates/updates.html.jinja2

```diff
@@ -22,7 +22,7 @@
       <a href="{{ project.url_for() }}" aria-label="{% trans %}Back to the project{% endtrans %}" class="mui--text-dark mobile-nav__icon" data-action="Back to the {{ project.title }}(videos page)">{{ faicon(icon='arrow-left', icon_size='title') }}</a><span class="mui--text-dark mobile-nav__headline">{% trans %}Updates{% endtrans %}</span>
     </div>
   </div>
-  {{ project_header(project, csrf_form, class='mui--hidden-xs mui--hidden-sm', current_page='updates', pinned=false) }}
+  {{ project_header(project, current_view.CsrfForm(), class='mui--hidden-xs mui--hidden-sm', current_page='updates', pinned=false) }}
```

Comment:
Baseframe already publishes a `csrf_token()` global. This parameter can be removed and replaced with a macro in Baseframe's `components.html.jinja2`. See: https://github.com/hasgeek/baseframe/blob/9be0dbe371854c29dec0b1d9cabf4550928795db/baseframe/templates/baseframe/mui/baseframe.html.jinja2#L24-L27